### ISSUE TITLE
Gedit

### ISSUE THREAD

**Issue by sacerd-OS** [catppuccin/catppuccin#224]
**Are you available to help in the creation of this port? 😄 (optional)**
Yes, already done.

**Additional context 📷 (optional)**
https://github.com/sacerd-OS/gedit

This is my first time contributing to open source. Let me know if I've screwed up.

**Comment by nekowinston** [comment]
Hi there! 👋 

Thanks a lot for your port, the repo looks good, but it's just missing a LICENSE file. Please get the [License](https://github.com/catppuccin/template/blob/main/LICENSE) from the template, if you intend to license it under MIT, like the rest of Catppuccin.

Once that is done, I'm happy to get it transferred. 👍

**Comment by sacerd-OS** [comment]
LICENSE added. Must've accidentally deleted it.
